config/environments/production.rb

```diff
@@ -14,6 +14,11 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
+  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
+  # `config/secrets.yml.key`.
+  config.read_encrypted_secrets = true
```

Comment:
As long as this doesn't *require* us to have these set up, that's fine, since I don't think they're there yet.

Reply:
The server seems to run fine without a `config/secrets.yml.key` file, so it looks like it's not required.